wireguard/wireguard_linux.go

```diff
@@ -93,6 +94,7 @@ func (nc *NCIface) ApplyAddrs() error {
 		}
 	}
 	for _, node := range config.GetNodes() {
+		log.Println("adding address to wg interface", node.Address, node.Address6)
```

Comment:
was this solved, or still required?

Reply:
not yet,  same ip is still assigned to all nodes